Content.Server/GameObjects/Components/Mobs/DamageThresholdTemplates/HumanTemplate.cs

```diff
@@ -42,23 +48,27 @@ public override HudStateChange ChangeHudState(DamageableComponent damage)
                         throw new System.InvalidOperationException(); //these should all be below the crit value, possibly going over multiple thresholds at once?
                     }
                     var modifier = totaldamage / (critvalue / normalstates); //integer division floors towards zero
-                    return new HudStateChange()
-                    {
-                        StateSprite = "Mob/UI/Human/human" + modifier.ToString() + ".png",
-                        effect = ScreenEffects.None
-                    };
+                    statusEffectsComponent?.Owner.SendNetworkMessage(statusEffectsComponent, new StatusEffectsMessage(StatusEffectsMode.Change,
```

Comment:
You shouldn't need to manually send network messages to work with these components here. They should provide a sane API that's less verbose and copy-pasta riddled, and keep the actual messages to themselves.

Also, using network messages only (instead of component states) means that this state will not get replicated to new clients correctly. So it's probably best if you just use component states.

Reply:
Yeah no wockers.

Reply:
I made a new commit using component states so lemme know if I did anything dumb.